.gitignore

```diff
@@ -0,0 +1,3 @@
+.github
+/target
+.vscode
```

Comment:
This should be setup at the host level. Otherwise you're gonna end up with lots of IDE specific ignore entries.

There are some tips here about how to do this: http://egorsmirnov.me/2015/05/04/global-gitignore-file.html

Reply:
good idea, thanks